src/os_kfbsd.c

```diff
@@ -144,9 +189,19 @@ static int kern_vfs_sleep(sqlite3_vfs *vfs, int microseconds) {
 }
 
 static int kern_vfs_current_time(sqlite3_vfs *vfs, double *pTime) {
+    static const sqlite3_int64 unixEpoch = 24405875*(sqlite3_int64)8640000;
```

Comment:
What are these values? Can we #define them?

Reply:
This is the same code as they use in their UNIX module.